Expired/Model/NotificationViewModel.swift

```diff
@@ -130,17 +134,19 @@ class NotificationViewModel: ObservableObject {
         product.removeFromNotifications(product.notifications!)
         save(context)
     }
-
     // Schedule a notification of a given product and its notification category (expiring-soon | expired)
```

Comment:
```suggestion

    // Schedule a notification of a given product and its notification category (expiring-soon | expired)
```

Add an empty line between functions